nautobot_golden_config/templates/nautobot_golden_config/configplan_create.html

```diff
@@ -41,7 +41,7 @@
 {% block buttons %}
 {% include "nautobot_golden_config/job_result_modal.html" %}
 
-<a href="#" class="openBtn" data-toggle="modal" data-target="#modalPopup">
+<a href="#" class="openBtn" data-toggle="modal" data-target="#modalPopup" data-backdrop="static" data-keyboard="false">
```

Comment:
If I understand correctly, `data-keyboard="false"` is for escape key, I am 50/50 if that is desirable. Perhaps part of the WIP journey, but just thinking out loud

Reply:
Yes, you are correct and I can leave it off. Please be aware, however, the escape key only seems to work when you first press `Tab` to hover over one of the buttons and then hit `Esc`. Maybe it's because we are still using bootstrap v3.